src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -20,6 +20,10 @@ public CommandResult(String feedbackToUser) {
         this.feedbackToUser = feedbackToUser;
         relevantPersons = null;
     }
+    
+    public String getFeedbackToUser(){
+    	return this.feedbackToUser;
```

Comment:
@potatowagon Mixing tabs and spaces.

FWIW I use the "show whitespace" options in eclipse to check for these stuff since the misconfigured autoindent can sometimes add tabs (http://stackoverflow.com/a/27828467)